src/utils/openInBrowser.ts

```diff
@@ -2,8 +2,9 @@ import { logger } from '@/logger';
 import { Navigation } from '@/navigation';
 import Routes from '@/navigation/routesNames';
 import { Linking } from 'react-native';
+import * as WebBrowser from 'expo-web-browser';
 
-export const openInBrowser = (url: string, internal = true) => {
+export const openInBrowser = (url: string, internal = true, SafariContext = false) => {
```

Comment:
Should keep variable name casing consistent, also this does something similar on Android, so a better name would be something like `useInAppBrowser`. Also given there are two "internal" contexts now it might be better align the `internal` name with something like `useDappBrowser`. Even better would be one variable that specifies the open location type, but it's not worth changing all the callers of this function to do that.